src/zeit/content/cp/centerpage.py

```diff
@@ -200,10 +200,7 @@ def _fill_area_cache(self):
     _area_teasered_content = gocept.cache.property.TransactionBoundCache(
         '_v_area_teasered_content', dict)
 
-    def is_teaser_present_above(self, current_area, content):
-        return content in self._teasered_content_above(current_area)
-
-    def _teasered_content_above(self, current_area):
+    def teasered_content_above(self, current_area):
```

Comment:
So wie ich es im Code sehe, könnte man das eigentlich ganz gut über Mocking lösen. Wenn man `_fill_area_cache` mocked und als Seiteneffekt `_cached_areas` befüllt, kann man zumindest `teasered_content_above` und `manual_content_below` gut testen. Statt ganzen `Area`s würde auch ein Mock / Test Objekt / Sentinel funktionieren, das einzige Attribut das benötigt wird ist `__name__`. Vielleicht eine `class FakeArea(object): __name__ = None` in der Test-Datei anlegen.

Offen bleibt dann noch, wie man `_fill_area_cache` testet. Aber da es hier ja um einen CenterPage Test geht, kann man denke ich ruhig `Region.values` order `Area.values` mocken. Dann geht das auch ganz gut.